### PR TITLE
feat: [BD-46] implement rtl for components from MailtoLink to Spinner

### PR DESCRIPTION
**Pull request by vlasovmichael**
- Implement RTL support for all component
- from MailtoLink to Spinner
- JIRA: https://openedx.atlassian.net/browse/PAR-646

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @vlasovmichael! I've created [BLENDED-1095](https://openedx.atlassian.net/browse/BLENDED-1095) to keep track of it in Jira.   More details are on the [BD-46](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3134685276/BD-46+Paragon+Blended+Support+and+Maintenance) project page.


When this pull request is ready, tag your edX technical lead.

File: src/Modal/marketing-modal.mdx [hunk 1]
```diff
@@ -24,8 +24,9 @@ The ``MarketingModal`` is a preconfigured `ModalDialog` that accepts an image an
   const [isOpen, open, close] = useToggle(false);
   return (
     <>
-      <Button variant="outline-primary" onClick={open}>Open marketing modal</Button>
-
+      <div className="d-flex">
```
Comment (adamstankiewicz) [inline src/Modal/marketing-modal.mdx:27]: [clarification] Why is it necessary to wrap this element with `div.d-flex`?

File: src/Layout/Layout.scss [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+h5,
+h6,
+.h1,
+.h2,
+.h3,
+.h4,
+.h5,
+.h6,
+p {
+    text-align: start;
```
Comment (adamstankiewicz) [inline src/Layout/Layout.scss:14]: I'm not sure we can safely add this property? As is, it breaks the headings on certain parts of the documentation site, for example (headings are no longer centered):

image

Comment (vlasovmichael) [reply src/Layout/Layout.scss]: You are right. removed

File: src/Modal/marketing-modal.mdx (hunk 1 above)
Comment (vlasovmichael) [reply src/Modal/marketing-modal.mdx:27]: [We can avoid it. d-flex needed only to move the button to the right side in RTL direction. The screen without d-flex](https://github.com/openedx/paragon/pull/1084#discussion_r810018519)

**Review by adamstankiewicz — APPROVED** [review]

**Comment by edx-webhook** [comment]
@vlasovmichael 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/paragon&entry.867055334=https://github.com/openedx/paragon/pull/1090&entry.1484655318=https://github.com/vlasovmichael&entry.752974735=2022-02-14+09:22&entry.1917517419=2022-02-25+12:07&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**